pubspec.yaml

```diff
@@ -21,7 +21,7 @@ dependencies:
     sdk: flutter
 
   # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
+  # Use with the CupertminoIcons class for iOS style icons.
```

Comment:
typo?

Reply:
oops..😅

Reply:
Can I be the contributor if possible ? I can help out with news feed for this project.

Reply:
hey sorry i haven't seen this message then. Yah sure you can become a contributor, feel free to add the news feature :)